mern-todo/client/src/components/App.jsx

```diff
@@ -0,0 +1,96 @@
+import { Component } from 'react';
+import ReactDOM from 'react-dom';
+import ListIterator from 'Components/ListIterator';
+import FormComponent from 'Components/FormComponent';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: [],
+    };
+    this.handleDelete = this.handleDelete.bind(this);
+    this.handleUserInput = this.handleUserInput.bind(this);
+  }
+
+  componentDidMount() {
+    axios.get('http://localhost:3000/todos')
+      .then((response) => {
+        this.setState({
+          items: response.data,
+        });
+        console.log(response);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }
+
+  handleDelete(elementToBeDeleted) {
+    const items = this.state.items.filter((item) => {
+      if (item._id !== elementToBeDeleted) {
+        return item;
```

Comment:
This is the most common mistake devs do while using filter method.
Check the doc https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter